dispatch/tests/apis.py

```diff
@@ -0,0 +1,103 @@
+from dispatch.vendor.apis import Facebook, FacebookAPIError
+from dispatch.tests.cases import DispatchAPITestCase
+from dispatch.tests.helpers import DispatchTestHelpers
+
+class FacebookTest(Facebook):
```

Comment:
I don't think you have inherit from the original api class -- `object` is probably enough